product_kosher/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Product Kosher
+==============
+
+This module adds product's kosher certificate management
+
+Installation
```

Comment:
This section can be removed - it's basically just standard install instructions, which could be different depending on env & isn't worth documenting individually.